src/components/VContentReport/VContentReportForm.vue

```diff
@@ -162,7 +164,7 @@ export default defineComponent({
       // Submit report
       try {
         await service.sendReport({
-          identifier: props.media.identifier,
+          identifier: props.media.id,
```

Comment:
:disappointed: Can't wait for TS to make these types of bugs a thing of the past for us :slightly_smiling_face: